packages/back-end/src/models/FactMetricModel.ts

```diff
@@ -31,16 +31,18 @@ export class FactMetricModel extends BaseClass {
     return this.context.hasPermission("readData", doc.projects || []);
   }
   protected canCreate(doc: FactMetricInterface): boolean {
-    return this.context.permissions.canCreateMetric(doc);
+    return this.context.permissions.canCreateFactMetric(doc);
   }
   protected canUpdate(
     existing: FactMetricInterface,
     updates: UpdateProps<FactMetricInterface>
   ): boolean {
-    return this.context.permissions.canUpdateMetric(existing, updates);
+    return this.context.permissions.canUpdateFactMetric(existing, {
+      projects: updates.projects || existing.projects,
```

Comment:
Is this needed?  Doesn't the update check already do this logic?

Reply:
Are you referring to defining the `projects` property for the second argument?

This was an attempt to be explicit since the `updates` in `canUpdate` might not contain the `projects` array property, but `canUpdateFactMetric` is expecting (currently) an object with the shape of Pick<FactMetricInterface, "projects">. And since `projects` is a required property on the `FactMetricInterface`, this made typescript happy.

I suppose, I could update the signature of `canUpdateFactMetric` to be something like:

```ts
public canUpdateFactMetric = (
    existing: Pick<FactMetricInterface, "projects">,
    updates: Partial<FactMetricInterface>
  ): boolean => {
    return this.checkProjectFilterUpdatePermission(
      existing,
      updates,
      "createMetrics"
    );
  };
```
And then I can just do: 
```ts
return this.context.permissions.canUpdateFactMetric(existing, updates);
```